torch/nested/_internal/nested_tensor.py

```diff
@@ -76,31 +98,16 @@ def __new__(
             False,
             True,  # dispatch_layout
             ks,
+            _allocate_storage=False,  # don't try to allocate storage based on non-zero size
```

Comment:
Don't you have a valid size/stride/offset values here? Why don't you want to allocate these? This is a fake storage for show and alias resolution, it doesn't actually use any memory ever.

Reply:
The storage size calculation fails with NestedInt sizes.

I didn't realize it was a fake storage though. Is it okay to have the zero-size storage used in this PR to avoid needing to deal with the NestedInt size or do you think there's other issues that will arise from that?

Reply:
I is better to have a storage, even if it is 0 size because it allows code that check for view-ness and stuff like that through storage to work.